pkg/security/ebpf/c/process.h

```diff
@@ -84,8 +84,8 @@ static struct proc_cache_t * __attribute__((always_inline)) fill_process_context
 
     // UID & GID
     u64 userid = bpf_get_current_uid_gid();
-    data->uid = userid >> 32;
-    data->gid = userid;
+    data->uid = userid;
```

Comment:
Nit: shall we explicitly use a mask to make sure to always retrieve the lower 32 bits?